eclipse-theia/theia-cpp-extensions
# [cpp] move the '@theia/cpp' extension

#### What it does


Fixes #62

- moves the `@theia/cpp` extension from the main repository.
- updates to the `@theia/cpp` documentation.
- updates to configurations for `mocha` tests.

#### How to test


- verify that the CI passes and tests are executed.
- perform a sanity check that the application runs.

#### Review checklist


#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

Signed-off-by: vince-fugnitto <vincent.fugnitto@ericsson.com>
